### DISCUSSION TITLE
4.7.8.Final

### DISCUSSION THREAD

**Discussion by jamezp** [resteasy/resteasy#3415] [Announcements]
## What's Changed
<h2>        Bug
</h2>
<ul>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-2845'>RESTEASY-2845</a>] -         ResteasyWebTarget.proxy(Class) problem with questionmark in `@Path` with Regex
</li>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-2874'>RESTEASY-2874</a>] -         Web Target Proxy not overriding content type header
</li>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-3177'>RESTEASY-3177</a>] -         ClientInvocation may not always submit requests asynchronously
</li>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-3217'>RESTEASY-3217</a>] -         MultipartFormDataOutput should use an ordered map instead of a HashMap
</li>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-3256'>RESTEASY-3256</a>] -         CDI managed beans do not inject `@Context` injection targets
</li>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-3286'>RESTEASY-3286</a>] -         CVE-2023-0482 RESTEasy: creation of insecure temp files
</li>
</ul>
        
<h2>        Task
</h2>
<ul>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-3283'>RESTEASY-3283</a>] -         Do not throw an exception for OPTIONS request
</li>
</ul>
                                                                                                            
<h2>        Component Upgrade
</h2>
<ul>
<li>[<a href='https://issues.redhat.com/browse/RESTEASY-3287'>RESTEASY-3287</a>] -         Upgrade Spring to 5.3.25
</li>
</ul>

**Full Changelog**: https://github.com/resteasy/resteasy/compare/4.7.7.Final...4.7.8.Final

<hr /><em>This discussion was created from the release <a href='https://github.com/resteasy/resteasy/releases/tag/4.7.8.Final'>4.7.8.Final</a>.</em>

**Comment by dovezp** [comment]
@jamezp, So CVE-2023-0482 was addressed following 4.7.8.Final? 

Why does https://github.com/advisories/GHSA-jrmh-v64j-mjm9 state that it affects <= 6.2.2.Final?

![image](https://github.com/user-attachments/assets/b65b06c1-62c5-42ae-9b91-72d6231e7838)

![image](https://github.com/user-attachments/assets/ca31b021-0ee2-43a5-9221-1d8a43e92853)

**Reply by dovezp** [reply]
@jamezp please update https://github.com/resteasy/resteasy/releases/tag/5.0.6.Final with disclosure of https://github.com/resteasy/resteasy/commit/4feb572249c292bb4861027674502c6b851fb132 instead of "[5.0] Back ported fixes by @jamezp in https://github.com/resteasy/resteasy/pull/3412"

**Reply by jamezp** [reply]
It's been updated. This was also fixed in 4.7.8.Final as well in https://github.com/resteasy/resteasy/pull/3413.

**Reply by jamezp** [reply]
Just an FYI I've also filed https://github.com/resteasy/resteasy/security/advisories/GHSA-2c6g-pfx3-w7h8.
